webpack.common.config.js

```diff
@@ -123,6 +123,7 @@ var resolve = {
         'addonHelper': staticPath('js/addonHelper.js'),
         'koHelpers': staticPath('js/koHelpers.js'),
         'addonPermissions': staticPath('js/addonPermissions.js'),
+        'addonSettings': staticPath('js/addonSettings.js'),
```

Comment:
Good to know. I'll eradicate these as I touch Javascript from now on.

:white_check_mark: